csharp/src/Microsoft.ML.OnnxRuntime/Training/NativeTrainingMethods.shared.cs

```diff
@@ -15,6 +15,7 @@ public struct OrtTrainingApi
             public IntPtr LoadCheckpoint;
             public IntPtr SaveCheckpoint;
             public IntPtr CreateTrainingSession;
+            public IntPtr CreateTrainingSessionFromArray;
```

Comment:
optional: `CreateTrainingSessionFromBuffer`?

Either is fine. I just like buffer more than array. But either works.

Reply:
I also prefer buffer over array, I changed it to array before pushing the code just to make this similar to Inference. I don't mind changing this to buffer. Depends how much we care for similarity in naming

Reply:
I think it is fine if names are not similar between training and inference.